extensions/amp-access-laterpay/0.1/amp-access-laterpay.css

```diff
@@ -0,0 +1,55 @@
+.amp-access-laterpay {
+  padding: 16px;
+  width: 420px;
```

Comment:
What would happen in, e.g., iPhone6 with `375px` width? E.g. do we need `max-width: 100%` here or something like that?